src/swap/changelly.js

```diff
@@ -199,9 +199,14 @@ export function makeChangellyPlugin(
       checkInvalidCodes(INVALID_CURRENCY_CODES, request, swapInfo)
 
       const fixedPromise = this.getFixedQuote(request, userSettings, opts)
-      // FIXME: Estimated swaps are temporarily disabled
-      const fixedResult = await fixedPromise
-      return fixedResult
+      const estimatePromise = this.getEstimate(request, userSettings, opts)
+      try {
+        const fixedResult = await fixedPromise
+        return fixedResult
+      } catch (e) {
+        const estimateResult = await estimatePromise
+        return estimateResult
+      }
```

Comment:
Not that it matters or anything, but this is kinda gross, this could have just be

```js 
try {
   return await this.getFixedQuote(request, userSettings, opts)
} catch (e) {
   return await this.getEstimate(request, userSettings, opts)
}
```
You don't really have to change it since it doesn't really matter, but I had to mention it ;)